apps/openmw/mwscript/scriptmanagerimp.cpp

```diff
@@ -118,6 +119,12 @@ namespace MWScript
                 }
 
                 mInterpreter.run (&iter->second.first[0], iter->second.first.size(), interpreterContext);
+                return true;
+            }
+            catch (const MissingImplicitRefError& e)
+            {
+                Log(Debug::Error) << "Execution of script " << name << " failed:";
+                Log(Debug::Error) << e.what();
```

Comment:
Log(Debug::Error) << "Execution of script " << name << " failed:\n" << e.what();

Reply:
It's a copy of the lines underneath... and there's a slightly different approach on line 70. Any preferences? Because it could at least all be the same in the same file.

Reply:
Actually, we use two log messages in row only in this file. By design, every log message is independent from others, and the message format can be changed in the future (to print error timestamp, for example).